test/integration/configuration/conf/test_auth_source_plugin.py

```diff
@@ -0,0 +1,76 @@
+import json
+import os
+import textwrap
+
+import pytest
+
+from conan.test.utils.file_server import TestFileServer
+from conan.test.utils.test_files import temp_folder
+from conan.test.utils.tools import TestClient
+from conans.util.files import save
+
+
+class TestErrorsAuthSourcePlugin:
```

Comment:
This class is named `TestErrorsAuthSourcePlugin` as the other one below, but I'm wondering if all the tests belonging to it are checking errors and not successful authentications 🤔